frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/uploadListItem.component.tsx

```diff
@@ -79,15 +88,19 @@ export const UploadListItem = ({
 			/>
 			<UploadListItemRevisionTag
 				control={control}
+				disabled={isUploading}
 				isSelected={isSelected}
 				errorMessage={errors.revisionTag?.message}
 			/>
-			<Button $selectedrow={isSelected} onClick={onClickEdit}>
-				<EditIcon />
-			</Button>
-			<Button $selectedrow={isSelected} onClick={onClickDelete}>
-				<DeleteIcon />
-			</Button>
+			<UploadProgress progress={progress} errorMessage={uploadErrorMessage} hidden={!isUploading} />
+			<span hidden={isUploading}>
```

Comment:
Would it be better to use a React optional rendering here to be consistent with other files?
```js
{!isUploading && (
	<Button $selectedrow={isSelected} onClick={onClickEdit}>
		<EditIcon />
	</Button>
	<Button $selectedrow={isSelected} onClick={onClickDelete}>
		<DeleteIcon />
	</Button>
}
```